be/src/storage/lake/vacuum.cpp

```diff
@@ -258,6 +259,31 @@ void run_clear_task_async(std::function<void()> task) {
 
 static Status collect_garbage_files(const TabletMetadataPB& metadata, const std::string& base_dir,
                                     AsyncFileDeleter* deleter, int64_t* garbage_data_size) {
+    // check if file exist or not, if not exist, skip remove it again
+    // use a simple method, pickup one file randomly and test exist or not
```

Comment:
files may be partially deleted success.
if the exist check is based on deleted file, other exist files can not be deleted?